spring-pulsar-boot-autoconfigure/src/main/java/org/springframework/pulsar/autoconfigure/PulsarAutoConfiguration.java

```diff
@@ -24,15 +24,17 @@
 import org.springframework.boot.context.properties.EnableConfigurationProperties;
 import org.springframework.context.annotation.Bean;
 import org.springframework.context.annotation.Import;
-import org.springframework.pulsar.core.DefaultPulsarConsumerFactory;
-import org.springframework.pulsar.core.DefaultPulsarProducerFactory;
 import org.springframework.pulsar.config.PulsarClientConfiguration;
 import org.springframework.pulsar.config.PulsarClientFactoryBean;
+import org.springframework.pulsar.core.DefaultPulsarConsumerFactory;
+import org.springframework.pulsar.core.DefaultPulsarProducerFactory;
 import org.springframework.pulsar.core.PulsarConsumerFactory;
 import org.springframework.pulsar.core.PulsarProducerFactory;
 import org.springframework.pulsar.core.PulsarTemplate;
 
 /**
+ * {@link org.springframework.boot.autoconfigure.EnableAutoConfiguration Auto-configuration} for Apache Pulsar.
```

Comment:
**[Nit/Question]** I wonder if it is sufficient to just link to [org.springframework.boot.autoconfigure.Autoconfiguration](/Users/cbono/.gradle/caches/modules-2/files-2.1/org.springframework.boot/spring-boot-autoconfigure/3.0.0-SNAPSHOT/af0fa2b33c020b4e6b128c54eadcbce0dcb48f47/spring-boot-autoconfigure-3.0.0-SNAPSHOT-sources.jar!/org/springframework/boot/autoconfigure/AutoConfiguration.java:64) now that its a 1st class thing in SB3? 

Also, does this have to be fully specified like it is?

Reply:
Sure, we can do it that way. Feel free to do that as part of the cleanup that you will do in the boot module.